packages/apiserver/src/file/file.service.ts

```diff
@@ -1,65 +1,77 @@
 import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
-import * as fs from 'fs';
-import * as path from 'path';
+import * as AWS from 'aws-sdk';
 import { v4 as uuidv4 } from 'uuid';
 
+const spacesEndpoint = new AWS.Endpoint(`${process.env['SPACE_ENDPOINT']}`);
```

Comment:
Why are you using direct access to env variables instead of `ConfigService`?

Reply:
i could use it too, but i saw many places where my teammates used it like i did, so i left it, should i change?

Reply:
I guess yes, but not in the scope of this PR. We need to create a task for refactoring